Use short “next review time” strings, fix plurals for time strings

ospalh opened the pull request:
This is the one change i promised a while ago.

* Fix plurals for time strings, for example French « 0 minute » instead of « 0 minutes ».
* Use unit symbols (s, min, d, mo, yr) in the ease (answer) buttons like Anki desktop.
* Those do not use plurals on purpose. Unit symbols are not abbreviations. See [BIPM](http://www.bipm.org/en/publications/si-brochure/section5-1.html).
* There is also a space between the time and the unit, again by the [book](http://www.bipm.org/en/publications/si-brochure/section5-3.html).
* This does not use the meter (“m”) as a unit of time, as Anki desktop did. (Maybe that has been fixed now.) A [meter](https://en.wikipedia.org/wiki/Grace_Hopper#Retirement) is a rather [short time](https://en.wikipedia.org/wiki/List_of_unusual_units_of_measurement#Light-nanosecond).
* Use the correct decimal separator – “.” or “,” – depending on the language selected. (This was a problem before only when you changed the app language to something other than the system language.)
* Some other, related clean up.

(edit: removed reference to French and German. [Lots of countries/languages](https://en.wikipedia.org/wiki/File:DecimalSeparator.svg) use a comma.)

ospalh commented:
I updated this and deleted one line from a docstring that applied to an intermediate version but made no sense for this commit.

timrae commented on AnkiDroid/src/main/java/com/ichi2/libanki/Utils.java:
```diff
@@ -103,16 +105,15 @@
+    private static final double TIME_HOUR = 60 * TIME_MINUTE;
+    private static final double TIME_DAY = 24 * TIME_HOUR;
+    private static final double TIME_YEAR = 31556930.0;  // seconds
+    // How long is a year? This is a tropical year, according to NIST,
```
Is there any reason why you put the comments below the variable they refer to? It's my first time to see that in somebody's code tbh.

ospalh replied:
I'm not sure why i started it.
Sure, i can change that.

timrae commented on AnkiDroid/src/main/res/values/01-core.xml:
```diff
@@ -30,7 +30,7 @@
     <string name="studyoptions_new_total">Total new cards:</string>
     <string name="studyoptions_total_cards">Total cards:</string>
-    <string name="studyoptions_eta">Estimated time (min):</string>
+    <string name="studyoptions_eta">Estimated time/min:</string>
```
I think the `/` would look weird as well... How about changing it to show the unit after the number?

ospalh replied:
You are right. it should have been "t/min" ...
But maybe going back to the old version is better. I like having only numbers on the right.

timrae replied:
Yeah I prefer the brackets over the forward slash to be honest

timrae commented on AnkiDroid/src/main/res/values/06-statistics.xml:
```diff
@@ -140,8 +106,8 @@
-        <item quantity="one">Studied %1$d card in %2$s today.</item>
-        <item quantity="other">Studied %1$d cards in %2$s today.</item>
+        <item quantity="one">Studied %1$d card in %2$s today</item>
+        <item quantity="other">Studied %1$d cards in %2$s today</item>
```
Could you please limit this PR to changes that break the Crowdin translations? I.e. adding / removing strings. Otherwise it makes it really hard to manage

ospalh replied:
Sure

ospalh replied:
Otoh, this is the text i wanted moved to `01-core.xml`, #3580, so that would break the translation anyway.
Should i move it now? Or remove the “.” again and open a new PR just moving the string? Do nothing more and leave it obscure?

timrae replied:
Let me take care of `stats_today_cards_not_bold`

timrae replied:
Sorry I changed my mind, please rename it and move it to core

ospalh commented:
I fixed the points mentioned.
While i was at it i also rebased it off the newest develop.
The non-squashed fixes are on my [feature-more-plurals-fixed](https://github.com/ospalh/Anki-Android/tree/feature-more-plurals-fixed) branch.

timrae commented:
Fixed #3580
